gcloud/pubsub/test__gax.py

```diff
@@ -376,3 +820,46 @@ class _ListTopicSubscriptionsResponsePB(object):
     def __init__(self, subscriptions, next_page_token=None):
         self.subscriptions = subscriptions
         self.next_page_token = next_page_token
+
+
+class _PushConfigPB(object):
+
+    def __init__(self, push_endpoint):
+        self.push_endpoint = push_endpoint
```

Comment:
@tseaver Couldn't we just use the actual proto classes? The only thing that we'd really be worried about stubbing is the network layer.